docs/topics/forms/modelforms.txt

```diff
@@ -525,9 +524,10 @@ the ``name`` field::
                 },
             }
 
-Finally, if you want complete control over of a field -- including its type,
-validators, etc. -- you can do this by declaratively specifying fields like you
-would in a regular ``Form``.
+.. versionadded:: 1.8
+
+You can also specify ``field_classes`` to customize the type of fields
+instanciated by the form.
```

Comment:
I think usually we try to describe the feature integrated into the main text, and then just add a versionadded note like "The `Meta.field_classes` argument was added" (and I think that text should actually be indented under the versionadded directive). This makes removing versionadded directives simpler later on.